Makefile

```diff
@@ -1,4 +1,5 @@
 SRCS=$(filter-out %_test.go, $(wildcard *.go */*.go))
+include integration/makefile.mk
```

Comment:
Q: Is there a reason why we do not directly include the `makefile.mk` integration commands in this Makefile?

Reply:
Maybe I misunderstood @ahrtr 's [comment](https://github.com/etcd-io/gofail/pull/69#discussion_r1611835042).

My intention is to keep all make commands related to integration tests in a dedicated makefile [1], so we can call it from the Makefile located in the root directory of the project [2].

Does this make sense to you? :) 

Reference: 
[1] https://github.com/etcd-io/etcd/blob/main/tests/robustness/makefile.mk
[2] https://github.com/etcd-io/etcd/blob/main/Makefile#L3

Reply:
Sure, makes sense. Thank you for the references!